CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `sandstone/Alert` and `sandstone/Input` support for portrait mode
+- `sandstone/Icon` supported icon list, adding a new icon `wallpaper`
+- `sandstone/VideoPlayer` props `backButtonAriaLabel` and `onBack` to provide a way to exit video player via touch
+
+### Changed
+
+- `sandstone/Panels.Header` and `sandstone/RadioItem` to use `onClick` instead of `onTap` for touch support
+- `sandstone/DatePicker` and `sandstone/TimePicker` to not show press effect via touch input
+- `sandstone/Scroller` and `sandstone/VirtualList` to show overscroll effect when flicking
```

Comment:
Don't we need the modification of useScroll/useOverscrollEffect.js?
https://github.com/enactjs/sandstone/pull/792/files

CC: @0x64

Reply:
@jeonghee27 we already have the logic change in [release/2.0.x branch](https://github.com/enactjs/sandstone/blob/release/2.0.x.develop/useScroll/useOverscrollEffect.js#L32).